.circleci/config.yml

```diff
@@ -63,11 +63,13 @@ commands:
           name: Verify codebase
           command: |
             cd MapboxSearch
+            ./gradlew :base:lint
             ./gradlew :sdk:lint
             ./gradlew :ui:lint
             ./gradlew :sdk-common:lint
             ./gradlew :autofill:lint
             ./gradlew :sample:lint
+            ./gradlew :base:lint
```

Comment:
why do we need to run same command again?

Reply:
Thank you, great catch! I'll fix this.